features/facebookImport/src/views/explore/ministory-styles.css

```diff
@@ -0,0 +1,14 @@
+.ministory-title {
+    max-width: 320px;
+    font-size: 24px;
+    font-weight: 500;
+    line-height: 28.8px;
+    margin: 0 0 15px 0;
+}
+
+.analysis-card .highlighted-number {
+    font-size: clamp(1rem, 15vw, 3.75rem);
```

Comment:
I just recently stumbled across clamp, quite the interesting css attribute I think.
(Min: 16px [is our base font-size], normal: full screen for at most 1 b., max: 60px)
Suggestions for value changes welcome. Take a look and tell me what you think @mteresa-jimenez @lenalaupoly

Reply:
That seems a good solution! 👏 👏 👏  
As you mentioned, the third number (3.75rem) is the max size. I've checked Figma and the size it's supposed to be `80px` (5rem), should the third number be that size then? (Not sure how the clamp attribute works or if that max size doesn't look good)

Reply:
That's right! I'll change that :)